### PR TITLE
Skanus receptai

### PR DESCRIPTION
**Pull request by AugustinasJarockis**
Naujas struct skirtas darbui su html kodu. Jo paskirtis: automatiskai uzdeti visus elementu atidarymus ir uzdarymus. Taip pat gali visa koda uzrasyti i faila. Strucktura paremta double linked list pagrindu. Turetu buti patogiau insertinti visus elementus. Please ignore the main.c faila, ten nesamone

### CODE REVIEW

**Review by arminasbrazenas — COMMENTED** [review]
We don't need most of these files. They are not required for our program to work (generated by CLion)

File: c stuff/HTML_code_doubly_linked_list.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    char *text;
+    struct HTML_CODE *previous;
+    struct HTML_CODE *next;
+} HTML_CODE;
+
+HTML_CODE *___HTML_START___ = NULL;
+HTML_CODE *___HTML_END___ = NULL;
+HTML_CODE *___HTML_CURRENT___ = NULL;
+
+void generate_HTML_file(char *file_name);
```
Comment (arminasbrazenas) [inline c stuff/HTML_code_doubly_linked_list.h:19]: We should to a coding standard (generate_html_code). Suggest [this](https://github.com/mcinglis/c-style).

File: c stuff/HTML_code_doubly_linked_list.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+HTML_CODE *create_HTML_CODE(char *text, HTML_CODE *previous, HTML_CODE *next);
+
+void insert_Field(char *name, char *param);
```
Comment (arminasbrazenas) [inline c stuff/HTML_code_doubly_linked_list.h:23]: don't need to uppercase if we use snake_case for function names (e.g. constants should be SNAKE_CASE)

Comment (arminasbrazenas) [reply c stuff/HTML_code_doubly_linked_list.h:23]: [suggestion](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841)

File: c stuff/HTML_code_doubly_linked_list.c [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    if(start_text == NULL)
+    {
+        printf("Failed to allocate memory for end_text\n");
+        return;
+    }
+    strcat(end_text, "</");
+    strcat(end_text, name);
+    strcat(end_text, ">");
+
+    if(___HTML_START___ != NULL)
```
Comment (arminasbrazenas) [inline c stuff/HTML_code_doubly_linked_list.c:83]: this name looks weird

File: c stuff/HTML_code_doubly_linked_list.h [hunk 4]
```diff
@@ -0,0 +1,30 @@
+//
+// Created by Augustinas on 2022-11-26.
+//
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+typedef struct HTML_CODE
```
Comment (arminasbrazenas) [inline c stuff/HTML_code_doubly_linked_list.h:8]: HTML_TAG maybe?

File: c stuff/cmake-build-debug/CMakeFiles/TargetDirectories.txt [hunk 5]
```diff
@@ -0,0 +1,3 @@
+C:/Users/jarau/Desktop/Reginos receptai/ReginosReceptai/c stuff/cmake-build-debug/CMakeFiles/c_stuff.dir
```
Comment (arminasbrazenas) [inline c stuff/cmake-build-debug/CMakeFiles/TargetDirectories.txt:1]: yep this shouldn't be here

File: c stuff/HTML_code_doubly_linked_list.h (hunk 4 above)
Comment (AugustinasJarockis) [reply c stuff/HTML_code_doubly_linked_list.h:8]: gal HTML_NODE? Nes čia viskas gali būti

File: c stuff/HTML_code_doubly_linked_list.c (hunk 3 above)
Comment (AugustinasJarockis) [reply c stuff/HTML_code_doubly_linked_list.c:83]: tas name yra field name

**Review by arminasbrazenas — APPROVED** [review]
God Bless This

**Pull request merged**